apps/twap/src/app/routes/_chainId/_walletAddress/orders/index.ts

```diff
@@ -0,0 +1,185 @@
+import { FastifyInstance } from 'fastify';
+import { Wallet } from '../../../../data/wallet';
+import { Order } from '../../../../data/order';
```

Comment:
Nipick: does it make sense to do full path imports to make it look cleaner?

Reply:
Can be a later follow up. Don't have capacity for this at the moment.